examples/recommendations/detect_and_apply_recommendations.rb

```diff
@@ -32,49 +36,88 @@ def detect_and_apply_recommendations(customer_id)
   # ENV['HOME']/google_ads_config.rb when called without parameters
   client = Google::Ads::GoogleAds::GoogleAdsClient.new
 
+  # [START detect_keyword_recommendations]
   query = <<~QUERY
-    SELECT recommendation.resource_name
+    SELECT recommendation.resource_name, recommendation.campaign,
+        recommendation.keyword_recommendation
     FROM recommendation
     WHERE recommendation.type = KEYWORD
-    LIMIT #{MAX_RESULT_SIZE}
   QUERY
 
-  reccomendation_service = client.service.recommendation
   google_ads_service = client.service.google_ads
-  NUMBER_OF_RUNS.times do |i|
-    search_response = google_ads_service.search(
-      customer_id: customer_id,
-      query: query,
-    )
-    operations = search_response.map do |row|
-      client.operation.apply_recommendation do |aro|
-        aro.resource_name = row.recommendation.resource_name
-      end
-    end
 
-    unless operations.empty?
-      response = recommendation_service.apply_recommendation(
-        customer_id: customer_id,
-        operations: operations,
-      )
+  response = google_ads_service.search(
+    customer_id: customer_id,
+    query: query,
+  )
 
-      response.each do |result|
-        puts "Applied recommendation with resource name #{result.resource_name}."
-      end
-    end
+  operations = response.each do |row|
+    recommendation = row.recommendation
+
+    puts "Keyword recommendation ('#{recommendation.resource_name}') was found for "\
+      "campaign '#{recommendation.campaign}'."
 
-    if i < NUMBER_OF_RUNS - 1
-      puts "Waiting #{PERIOD_IN_SECONDS} seconds before applying more recommendations."
-      sleep(PERIOD_IN_SECONDS)
+    if recommendation.keyword_recommendation
+      keyword = recommendation.keyword_recommendation.keyword
+      puts "\tKeyword = '#{keyword.text}'\n\ttype = '#{keyword.match_type}'"
```

Comment:
Can you split this into two puts statements (one per line) instead of having \n in the middle? I think that makes it more readable.

Reply:
done